python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -57,6 +58,66 @@
 ]
 
 
+def calculate_apparent_motion(data, refEpoch):
+    """Calculate shift from reference epoch to the apparent observed position
+    at another date.
+
+    This function calculates the shift due to proper motion combined with the
+    apparent motion due to parallax. This is not used in the
+    `GbdesAstrometricFitTask` or related child tasks, but is useful for
```

Comment:
I don't mind doing that, except that this is specifically geared to match what is in `gbdes`. It is calculating the motion relative to other stars in the field, not the actual apparent motion, which would also include annual aberration. I am also using things like the ecliptic angle hard-coded in `gbdes` rather than one of the astropy options to better match what is coming out of the fitter.